proxies/nodejs/app.ts

```diff
@@ -1,14 +1,24 @@
 import { initialize } from '@devcycle/nodejs-server-sdk'
 import Koa from 'koa'
+import Router from 'koa-router'
+
 
 async function start() {
     await initialize('<TOKEN>').onClientInitialized()
 
     const app = new Koa()
 
-    app.use(async (ctx) => {
-      ctx.body = 'Hello World'
+    var router = Router()
+    router.get('/spec', (ctx) => {
+      ctx.status = 200
+      ctx.body = {
+        name: "NodeJS",
+        version: "", // TODO add branch name or sdk version here
+        capabilities: ["EdgeDB", "SSE", "LocalBucketing"],
```

Comment:
technically SSE is not a capability of the nodejs sdk (or any serverside sdk)

Reply:
oops right, fixed!